catalog/views.py

```diff
@@ -1,7 +1,7 @@
 import datetime
 
 from django.views import generic
-from django.shortcuts import render,get_object_or_404
+from django.shortcuts import render,get_object_or_404,redirect
```

Comment:
thêm khoảng trắng giữa các method
```suggestion
from django.shortcuts import render, get_object_or_404, redirect
```